packages/terra-application/src/page/PageActions.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/* eslint-disable react/no-unused-prop-types */
+const propTypes = {
+  /**
+   * The PageAction elements to render for the Page.
+   */
+  children: PropTypes.node,
+};
+/* eslint-enable react/no-unused-prop-types */
+
+/**
+ * The PageActions component allows Page consumers to define visible actions
+ * using JSX. The Page will parse props from  PageActions and present a
+ * variety of controls for the actions depending on Page constraints.
+ *
+ * The PageActions component should not be rendered directly.
+ */
+const PageActions = () => <div />;
```

Comment:
it doesn't matter, but could this be an empty fragment?

Reply:
Good question. It needs to be non-null so that the props table parsing will recognize it as a component, but I imagine a fragment would work too? I'll try it out.

Reply:
It works!